aiokafka/protocol/produce.py

```diff
@@ -190,20 +183,27 @@ class ProduceRequest_v0(ProduceRequest):
         ),
     )
 
+    required_acks: int
 
-class ProduceRequest_v1(ProduceRequest):
+    def expect_response(self) -> bool:
```

Comment:
What is the reason for moving it from base class?

Reply:
Because the base class doesn't contain `required_acks` in the schema.

Reply:
Do I understand correctly, that all subclasses of `ProduceRequestBase` have `required_acks`? If so, both `required_acks` and `expect_response` redefinition can be places there.

Reply:
Ok, done